src/components/footer/config.ts

```diff
@@ -0,0 +1,333 @@
+import { Brand } from '../../types/brand';
+
+interface VisibilitySettings {
+  brand: {
```

Comment:
could we extract the brand to a type/enum and use it like
```typescript
type VisibilitySettings = { brand: Record<Brand, boolean> }
```
just to keep it dynamic. Also, this Brand enum could then be shared with the projects and the themeprovider